web/html/src/components/input/Select.js

```diff
@@ -1,12 +1,23 @@
 // @flow
 
 import * as React from 'react';
+import { default as ReactSelect } from 'react-select';
```

Comment:
Would recommend `import ReactSelect from 'react-select';` instead unless there's a specific reason for importing this way here?

Reply:
The real thing to import is named `Select` and that would conflict with the name of the component that is exported in this JS module... The regular import is `import Select from 'react-select';` but maybe I'm overlooking something.

Reply:
Unless we have a linter rule that disallows it, importing the default export is already an alias, even without `as`. To shortly elaborate, all of these should be equivalent:  
```ts
import Foo from 'somewhere';
const Bar = Foo;

import { default as Bar } from 'somewhere';

import Bar from 'somewhere';
```

The main benefit of aliasing imports with `as` is when you want to rename named exports:  
```ts
import { tea as coffee } from 'somewhere';
```

Reply:
addressed